spec/models/occupation_standard_spec.rb

```diff
@@ -199,6 +199,29 @@
     end
   end
 
+  describe ".recently_added" do
+    it "excludes occupations without work processes" do
+      create_list(:occupation_standard, 4)
```

Comment:
Can we just create 2 objects here instead of 4? It will serve the same purpose and reduces DB insert count.